### DISCUSSION TITLE
"Client TLS handshake failed." even after patching it.

### DISCUSSION THREAD

**Discussion by llama-for3ver** [mitmproxy/mitmproxy#6469] [Intercept Everything!]
I'm trying to debug an Android app blah blah blah but even after patching it w/ apk-mitm (and a bit with Frida) I'm still getting the following error in my console:
`Client TLS handshake failed. The client disconnected during the handshake. If this happens consistently for [domain], this may indicate that the client does not trust the proxy's certificate.`

This only happens for specific endpoints.
How should I continue on patching it?

**Comment by 3052** [comment]
I found recently that MITM Proxy is failing with Frida, but if you use the same exact script with HTTP Toolkit it works. I prefer MITM Proxy, but if it doesn't work I have no choice but to use something else

**Reply by llama-for3ver** [reply]
I found out that the app was using OkHTTP pinning which apk-mitm wasn't patching but I'll try toolkit.

**Reply by llama-for3ver** [reply]
@3052 It actually worked... I've been trying for weeks to get it to work. TYSM!!
